### PR TITLE
search suggest - make case insensitive

### PR DESCRIPTION
**Pull request by YuriHechter**

### CODE REVIEW

File: src/helpers/search.js [hunk 1]
```diff
@@ -48,10 +48,10 @@ export class SuggestionsHelper {
         if (a.part !== b.part) {
           return a.part - b.part;
         }
-        if (a.suggest.startsWith(query) && !b.suggest.startsWith(query)) {
+        if (a.suggest.toLowerCase().startsWith(query) && !b.suggest.toLowerCase().startsWith(query)) {
```
Comment (gshilin) [inline src/helpers/search.js]: toLowerCase is potentially called for multiple times for every suggestion.
Probably it's better to map them to lower case before sort()?

File: src/helpers/search.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ export class SuggestionsHelper {
     this.suggestions = [];
 
     if (results && results.suggest && 'title_suggest' in results.suggest) {
```
Comment (gshilin) [inline src/helpers/search.js]: results && results.suggest && results.suggest['title_suggest']

File: src/helpers/search.js (hunk 1 above)
Comment (YuriHechter) [reply src/helpers/search.js]: changed

File: src/helpers/search.js (hunk 2 above)
Comment (YuriHechter) [reply src/helpers/search.js]: done

**Comment by bbfsdev** [comment]
Reopening to master.

**Pull request closed without merging**